packages/web3/src/utils/exchange.test.ts

```diff
@@ -24,7 +24,7 @@ import {
   getSenderAddress,
   getALPHDepositInfo,
   isSimpleALPHTransferTx,
-  isSimpleTransferTokenTx,
+  isSimpleTokenTransferTx,
```

Comment:
How about removing these two `isSimple*` functions? They are not recommended for exchanges now.